package.json

```diff
@@ -34,7 +34,8 @@
         "default": "./lib/generator.js"
       },
       "deno": "./dist/generator-deno.js",
-      "default": "./dist/generator.js"
+      "default": "./dist/generator.js",
+      "types": "./lib/generator.d.ts"
```

Comment:
`"types"` should always go first as the first export item in the list.

Reply:
Done